packages/legend-application-studio/src/stores/ChangeDetectionState.ts

```diff
@@ -335,12 +339,29 @@ export class ChangeDetectionState {
     this.observerContext = new ObserverContext(
       this.editorStore.pluginManager.getPureGraphManagerPlugins(),
     );
+
+    this.graphHash = hashArray(
+      this.editorStore.graphManagerState.graph.allOwnElements,
+    );
   }
 
   setAggregatedProjectLatestChanges(diffs: EntityDiff[]): void {
     this.aggregatedProjectLatestChanges = diffs;
   }
 
+  snapshotGraphHash(): void {
```

Comment:
therefore, we don't need this method, whoever need to get the current graph hash, just call `getCurrentGraphHash`